plugin/src/main/java/org/opensearch/ml/task/MLTaskManager.java

```diff
@@ -384,4 +389,39 @@ public boolean containsModel(String modelId) {
         }
         return false;
     }
+
+    public String[] getLocalRunningLoadModelTasks() {
+        List<String> runningLoadModelTaskIds = new ArrayList<>();
+        for (Map.Entry<String, MLTaskCache> entry : taskCaches.entrySet()) {
+            MLTask mlTask = entry.getValue().getMlTask();
+            if (mlTask.getTaskType() == MLTaskType.LOAD_MODEL && mlTask.getState() != MLTaskState.CREATED) {
+                runningLoadModelTaskIds.add(entry.getKey());
+            }
+        }
+        return runningLoadModelTaskIds.toArray(new String[0]);
+    }
+
+    public void syncRunningLoadModelTasks(Map<String, Set<String>> runningLoadModelTasks) {
+        Instant ttlEndTime = Instant.now().minus(120, ChronoUnit.SECONDS);
```

Comment:
Is this 2 mins too short? If possible, why not use a longer time? It does no harm to keep a task for 10 mins?

Reply:
Yes, good point. I used 2 minutes for quick testing. I think we do need to increase to a longer TTL. 10 mins looks good. Will change this part in next PR.